src/patterns/webform/examples/success/index.njk

```diff
@@ -0,0 +1,36 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/button/_macro.njk" import onsButton %}
+
+{% set pageConfig = {
+    "title": 'Thank you for contacting us - Census 2021',
+    "theme": 'census',
+    "absoluteUrl": pageInfo.url,
+    "headMeta": {
+        "description": 'Census 2021 is a UK wide population survey'
+    },
+    "header": {
+        "logoHref": '#0',
+        "titleLogo": 'census-logo-en',
+        "titleLogoHref": '#0',
+        "titleLogoAlt": 'Census 2021'
+    },
+    "footer": {
+    }
+} %}
+{% block main %}
+    {{
```

Comment:
I think the success page probably needs CTA as a route onwards for the user.  Either a breadcrumb to 'Home', or a CTA button.  We used 'Done' at the end of all Census RH fulfilment journeys.

Reply:
I will match the 'Done' as in the Census RH fulfilment journeys.